libs/components/forms/src/lib/modules/file-attachment/file-attachment/file-attachment.component.html

```diff
@@ -167,7 +167,7 @@
 
 <sky-form-errors
   [id]="errorId"
-  [errors]="ngControl?.errors ?? fileErrorValidation"
```

Comment:
we handle all the errors inside `fileErrorValidation` ourselves so we don't need to be sending this to `sky-form-errors`